astropy/io/fits/tests/test_checksum.py

```diff
@@ -205,9 +205,9 @@ def test_ascii_table_data(self):
                 # The checksum ends up being different on Windows, possibly due
                 # to slight floating point differences
                 assert 'CHECKSUM' in hdul[1].header
-                assert hdul[1].header['CHECKSUM'] == '51IDA1G981GCA1G9'
+                assert hdul[1].header['CHECKSUM'] == '3rKFAoI94oICAoI9'
                 assert 'DATASUM' in hdul[1].header
-                assert hdul[1].header['DATASUM'] == '1948208413'
+                assert hdul[1].header['DATASUM'] == '1914653725'
```

Comment:
Note that this change is needed because the file has a column with scale, so the E->D changes the checksums.

Reply:
Does this mean it's backwards-incompatible? I mean it would be bad if checksum or datasum tests for existing tests failed.

Reply:
You mean for a package that would be testing the computed checksum ? The content of the file changes, so the checksum as well. I can be problematic in some cases, for sure, but on the other hand it fixes a possible loss of precision for other software.

Reply:
I was thinking about verifying a file with an existing checksum.

Reply:
I had a look more in detail, and it works. For "old" files, saved before this PR, the computed checksum does not change. This is because the checksum is always computed on the raw bytes.